spec.html

```diff
@@ -13476,7 +13393,7 @@ <h1>
     <emu-clause id="sec-createbuiltinfunction" type="abstract operation">
       <h1>
         CreateBuiltinFunction (
-          _behaviour_: unknown,
+          _behaviour_: an Abstract Closure, a set of algorithm steps, or some other definition of a function's behaviour provided in this specification,
```

Comment:
This preamble now reads

> The abstract operation CreateBuiltinFunction takes arguments behaviour (an Abstract Closure, a set of algorithm steps, or some other definition of a function's behaviour provided in this specification), length (a non-negative integer or +∞), name (a property key), and additionalInternalSlotsList (a List of names of internal slots) and optional arguments realm (a Realm Record), prototype (an Object or null), and prefix (a String).

I... guess that's ok? But it's getting pretty hard to read.

Reply:
I think it reads fine. It's the number of parameters that's the issue, not the description of their types.